msi/release.sh

```diff
@@ -206,12 +206,26 @@ EOF
 mkdir -p obj &&
 wix/candle.exe -dVersion="${VERSION%%-*}" \
 	-arch $WIX_ARCH \
-    GitProduct.wxs GitComponents.wxs -o obj\\ \
-	-ext WixUtilExtension -ext WixTagExtension &&
+    GitProduct.wxs GitComponents.wxs \
+    GitChooseEnvironmentDlg.wxs GitChooseTerminalDlg.wxs \
+    GitCustomizeLineEndingsDlg.wxs GitLineEndingsDlg.wxs \
+    GitMaintenanceTypeDlg.wxs GitPerformanceOptionsDlg.wxs \
+    -o obj\\ \
+    -ext WixUtilExtension -ext WixTagExtension \
+    -ext WixUIExtension &&
 wix/light.exe \
 	obj/GitProduct.wixobj \
     obj/GitComponents.wixobj \
+	obj/GitChooseEnvironmentDlg.wixobj \
+	obj/GitChooseTerminalDlg.wixobj \
+	obj/GitCustomizeLineEndingsDlg.wixobj \
+	obj/GitLineEndingsDlg.wixobj \
+	obj/GitMaintenanceTypeDlg.wixobj \
+	obj/GitPerformanceOptionsDlg.wixobj \
 	-o $TARGET -ext WixUtilExtension -ext WixTagExtension \
+	-ext WixUIExtension -loc GitUI_en-us.wxl \
+	-cultures:en-us \
+	-spdb \
```

Comment:
Ahh, perfect. Now I now where to find a repro of that bug then. I'll track down the WiX v4.0 bug after you get this checked-in so we can get .wixpdbs back.